### DISCUSSION TITLE
能否加入评分信息？

### DISCUSSION THREAD

**Discussion by Simon7612** [zhaohongxuan/obsidian-weread-plugin#199] [Q&A]
微信读书本身就有评分，能否加入rating的值
![image](https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/88783407/ff992636-06a1-4e50-a50a-aadcf66c0875)

**Comment by Simon7612** [comment]
找到方法了，在模板 **# 元数据** 中加入
> ```
> > - 推荐值：{{metaData.rating}}
> ```

**Reply by palwoo** [reply]
这个是综合推荐值。您知道个人对书的打分值用什么表示吗？网上好像查不到微信读书所有的元数据

**Reply by palwoo** [reply]
@zhaohongxuan 不知道书籍的元数据中对个人给书的打分用什么表示？您那里掌握的有吗？在网上的公开信息中没有查到。这个比起rating综合评分来说对个人应该更有价值。

**Reply by zhaohongxuan** [reply]
个人的打分只有三个：推荐、一般、不行，应该有的，我找找看
